aggregators/internal/telemetry/metrics.go

```diff
@@ -69,56 +68,40 @@ func NewMetrics(provider pebbleProvider, opts ...Option) (*Metrics, error) {
 	meter := cfg.Meter
 
 	// Aggregator metrics
-	i.RequestsTotal, err = meter.Int64Counter(
-		"aggregator.requests.total",
-		metric.WithDescription("Total number of aggregation requests"),
+	i.RequestsCount, err = meter.Int64Counter(
+		"aggregator.requests.count",
```

Comment:
[**For reviewers**] This metric is different from the `events.processed.count` since it tracks consumption from the queue. We can remove this entirely since we should be able to get this metric from `apm-queue`'s production/consumption metric. Let me know if that makes sense and I can drop this metric.

Reply:
Reference to relevant apm-queue metrics: https://github.com/elastic/apm-queue/blob/d13869f9c9c08705cc78aec24556dadc5ee8ec42/kafka/metrics.go#L39-L42

Reply:
@axw WDYT about removing the `aggregator.requests.count` metric? For context, this metric aims to track the number of requests (one request can have multiple events) received by `Aggregate*` methods. I think, we should be good with `consumer.messages.fetched` metric that is produced by apm-queue and remove `aggregator.requests.count`.

Reply:
The consumer part makes sense for `aggregation-service`, for `apm-server`, the concept of aggregation request would be confusing I think.

Reply:
Removing `aggregator.requests.count` SGTM

Reply:
Dropped `aggregator.requests.count`